docs/eventing/broker/README.md

```diff
@@ -57,6 +57,56 @@ spec:
         name: dlq-service
 ```
 
+### Creating a broker by annotation
+
+You can create a broker annotate your namespace:
+
+```shell
+kubectl label namespace <namespace> knative-eventing-injection=enabled
+```
+
+This will automatically create a broker named `default`, which is configured to use
+Kafka channels, in the specified namespace.
```

Comment:
unless the default-br-channel has been somewhere else to be configured to use Kafka as default, then this is not exactly right?
So, here we show how to configure to use Kafka channels for a certain namespace (test-broker-6, cause that's what I was using for my testing :) )
https://github.com/knative/docs/blob/master/docs/eventing/broker/mt-channel-based-broker.md

So, maybe just drop the Kafka channel here, and add another section that specifically talks about how to configure default channels?

So, out of the box, simple example, uses, say IMC as the default channel. Then having a different section for configuring the defaulting behaviour?

Also, given how easy it is now to create brokers, I'm wondering if we should first just have a user create a Broker using a kubectl?

Reply:
I would say this is all stuff we should follow up in our subsequent PRs to fix up this section - there's a lot of work to do to fix channels too, period.